src/main/resources/languages/en/messages.yml

```diff
@@ -174,6 +174,7 @@ messages:
     select: 'Select this recipe'
     temporarily-disabled: '&eThis Auto-Crafter was temporarily disabled. You can re-enable it at any time!'
     re-enabled: '&eThis Auto-Crafter was re-enabled!'
+    recipe-disabled: '&cThe recipe of this item has been disabled on this server.'
```

Comment:
I don't think this message is verbose or accurate enough.
You are checking for a specific gamerule and whether the recipe is discovered.
This does not necessarily imply that the recipe is disabled.
I will think of a better message and comment.

Reply:
Okay. Would "You don't have permission to use the recipe of this item" be good?